src/Infra/HttpClients/NotificationHttpClient.cs

```diff
@@ -39,7 +39,7 @@ public async Task<HttpResponseMessage> CreateNotification(string token, int subs
 
         string jsonPayload = System.Text.Json.JsonSerializer.Serialize(payload);
         HttpContent content = new StringContent(jsonPayload, Encoding.UTF8, "application/json");
-        var result = await _httpClient.PostAsync($"{_httpClient.BaseAddress}Notification", content);
```

Comment:
mesmo comentário do notification, não seria necessário fazer isso já que o baseadress está sendo atribuido na program e estamos utilizando http client factory, talvez o seu app settings esteja diferente e por isso não conseguiu atribuir, lembrando que http ou https pode diferenciar na hora de desenvolver caso o certificado não tenha sido criado para o SSL quando for https

Reply:
Como conversamos, o BaseAdress não está sendo setado aqui, só repassado para acessar o endpoint